src/instructlab/model/chat.py

```diff
@@ -172,6 +176,7 @@ def is_openai_server_and_serving_model(
     "--temperature",
     cls=clickext.ConfigOption,
 )
+@rag_options
```

Comment:
so we are implicitly passing these in?

the way we do this with config entries is map the nested classes to click.options where the defaults are funneled in via `cls=clickext.ConfigOption`. I would prefer to 

1. make the config options a class not a dictionary 
2. add associated flags to override in `ilab model chat` where the defaults use the above class `ConfigOption` to read defaults from the config

Reply:
1. ok, as you also requested in another comment
2. sure, I'll use the regular design then